client/test/e2e/debtors/debtorgroups.spec.js

```diff
@@ -0,0 +1,65 @@
+/* global browser, element, by, protractor */
+var chai = require('chai');
+var expect = chai.expect;
+
+var helpers = require('../shared/helpers');
+helpers.configure(chai);
+
+var components = require('../shared/components');
+var FU = require('../shared/FormUtils');
+
+describe('Debtor Groups Management', function () {
+  'use strict';
+
+
+  /** @const */
+  var root = '#/debtors/groups';
+
+  before(function () { browser.get(root); });
+
+  it('lists base test debtor groups', function () {
+
+    expect(element.all(by.css('[data-group-entry]')).count()).to.eventually.equal(2);
```

Comment:
Note for future reference - we should really make a utility class to do this.  Something like `FU.count(selector, n)`.